test/integration/health_check_integration_test.cc

```diff
@@ -669,6 +669,15 @@ TEST_P(TcpHealthCheckIntegrationTest, DisableHCForActiveTraffic) {
 
   EXPECT_EQ(1, test_server_->counter("cluster.cluster_1.health_check.success")->value());
   EXPECT_EQ(0, test_server_->counter("cluster.cluster_1.health_check.failure")->value());
+
+  timeSystem().advanceTimeWait(std::chrono::seconds(10));
+
+  test_server_->waitForCounterEq("cluster.cluster_1.health_check.attempt", 3);
+  result = clusters_[cluster_idx].host_fake_raw_connection_->write("Pong");
+  RELEASE_ASSERT(result, result.message());
```

Comment:
This doesn't actually check that you get the ping though, right? Can we actually check that?

/wait

Reply:
fixed